src/components/Books.js

```diff
@@ -1,7 +1,6 @@
-/* eslint-disable import/extensions */
 import React from 'react';
-import MyBook from './bookItem.js';
-import Form from './bookForm.js';
+import MyBook from './BookItem';
+import Form from './BookForm';
 
```

Comment:
- [OPTIONAL ] Destructuring `props` Instead of accessing `book.title` and `book.author`, you can destructure the book object in the map() function like this: `{title, author}`. This will make your code more concise and easier to read. :wink:
check this on lines 19 and 20
 ```{Books.map((book) => (
  <div key={`${book.title}-${book.author}`} className="book-item">
    ...
  </div>
))}```